### ISSUE TITLE
aria-valid-attribute: Ensures all ARIA attributes have valid values

### ISSUE THREAD

**Issue by corinagum** [microsoft/BotFramework-WebChat#3811]
Files as P0 due to deadline; as a need to fix it is P1
## Version
Related to https://github.com/microsoft/BotFramework-WebChat/pull/3098

## Describe the bug

`aria-labelledby` should be removed from hidden element

## Steps to reproduce

1. Open MockBot
2. Type/send 'carousel'
3. Use Fastpass and see that `aria-labelledby` is invalid

## Expected behavior

`aria-labelledby` should not be a present attribute

[Bug]
